pennylane/qchem/factorization.py

```diff
@@ -237,7 +239,7 @@ def basis_rotation(one_electron, two_electron, tol_factor=1.0e-5):
         .. math::
 
             H = \sum_{\alpha \in \{\uparrow, \downarrow \} } \sum_{pq} T_{pq} a_{p,\alpha}^{\dagger}
-            a_{q, \alpha} + \frac{1}{2} \sum_r^R \left ( \sum_{\alpha, \beta \in \{\uparrow, \downarrow \} } \sum_{pq}
+            a_{q, \alpha} + \frac{1}{2} \sum_r^R \left ( \sum_{\alpha \in \{\uparrow, \downarrow \} } \sum_{pq}
```

Comment:
When I build the docs locally or look at buildthedocs [here](https://xanaduai-pennylane--3714.com.readthedocs.build/en/3714/code/api/pennylane.qchem.factorize.html) the equations seem different from what is written here. Am I looking in the wrong place or what is going on?

Reply:
You may want to look [here](https://xanaduai-pennylane--3714.com.readthedocs.build/en/3714/code/api/pennylane.qchem.basis_rotation.html).